From c3a2e8aef84d037ef51e8b51d99ea15dad4cc2ff Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:40:13 -0400
Subject: [alsa] riptide: fix %d confusingly prefixed with 0x in format strings

Message-id: <140972823573-408-patchset-jkysela@redhat.com>
Patchwork-id: 91104
O-Subject: [RHEL 7.1 PATCH 407/442] ALSA: riptide: fix %d confusingly prefixed with 0x in format strings
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

Signed-off-by: Hans Wennborg <hans@hanshq.net>
Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: 47efe636adec37d6c259a69ee53d246cd4bba172)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/riptide/riptide.c b/sound/pci/riptide/riptide.c
index b4a8278..f0315c3 100644
--- a/sound/pci/riptide/riptide.c
+++ b/sound/pci/riptide/riptide.c
@@ -941,7 +941,7 @@ setmixer(struct cmdif *cif, short num, unsigned short rval, unsigned short lval)
  union cmdret rptr = CMDRET_ZERO;
  int i = 0;
 
- snd_printdd("sent mixer %d: 0x%d 0x%d\n", num, rval, lval);
+ snd_printdd("sent mixer %d: 0x%x 0x%x\n", num, rval, lval);
  do {
   SEND_SDGV(cif, num, num, rval, lval);
   SEND_RDGV(cif, num, num, &rptr);
@@ -1080,7 +1080,7 @@ getmixer(struct cmdif *cif, short num, unsigned short *rval,
   return -EIO;
  *rval = rptr.retwords[0];
  *lval = rptr.retwords[1];
- snd_printdd("got mixer %d: 0x%d 0x%d\n", num, *rval, *lval);
+ snd_printdd("got mixer %d: 0x%x 0x%x\n", num, *rval, *lval);
  return 0;
 }
 
-- 
1.7.1